engine/verification/test/verification_test.go

```diff
@@ -150,7 +150,7 @@ func TestSingleCollectionProcessing(t *testing.T) {
 	err = verNode.FinderEngine.Process(exeIdentity.NodeID, completeER.Receipt)
 	assert.Nil(t, err)
 
-	unittest.RequireReturnsBefore(t, conWG.Wait, 5*time.Second, "consensus nodes process")
+	unittest.RequireReturnsBefore(t, conWG.Wait, 30*time.Second, "consensus nodes process")
```

Comment:
Tests keep failing if this is too low. I need to check whats up with that before merging!